content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst

```diff
@@ -4,34 +4,118 @@ Inventory adjustments
+due to damages, human errors, theft, or other factors. As such, inventory adjustments must be made
 to reconcile the differences, and ensure that the recorded counts in the database match the actual
 counts in the warehouse.
 
-In Odoo, inventory adjustments can be accessed through :menuselection:`Inventory --> Operations -->
-Inventory Adjustments`.
+Inventory adjustments page
+==========================
 
-The :guilabel:`Inventory Adjustments` page shows all products that are currently in stock. Each
-line contains the following information:
+To view the :guilabel:`Inventory Adjustments` page, navigate to :menuselection:`Inventory app -->
+Operations --> Inventory Adjustments`.
 
-- :guilabel:`Product`
-- :guilabel:`Lot/Serial Number`
-- :guilabel:`On Hand Quantity`
+On the :guilabel:`Inventory Adjustments` page, all products that are currently in stock are listed,
+with each product line containing the following information:
+
+.. image:: count_products/count-products-inventory-adjustments-page.png
+   :align: center
+   :alt: Inventory adjustments page.
+
+- :guilabel:`Location`: the specific location in the warheouse where a product is stored.
+- :guilabel:`Product`: the product whose quantity is listed on the inventory adjustment line.
+- :guilabel:`Lot/Serial Number`: the specific tracking identifier assigned to the specific product
+  listed. It can contain letters, numbers, or a combination of both.
+
+.. note::
+   If a specific product has a quantity of more than `1.00` in stock, and more than one serial
+   number (or lot number) assigned to it, each uniquely-identified product is displayed on its own
+   product line with its own lot/serial number, displayed under the :guilabel:`Lot/Serial Number`
+   column.
+
+- :guilabel:`On Hand Quantity`: the quantity of the product currently recorded in the database.
+- :guilabel:`UoM`: the *unit of measure* in which the product is measured. Unless otherwise
+  specified (i.e., in **Pounds** or **Ounces**), the default :abbr:`UoM (unit of measure)` is
+  **Units**.
+- :guilabel:`Counted Quantity`: the real quantity counted during an inventory count. This field is
+  left blank by default, but can be changed, depending on if it matches the :guilabel:`On Hand
+  Quantity` or not.
+- :guilabel:`Difference`: the difference between the :guilabel:`On Hand Quantity` and
+  :guilabel:`Counted Quantity`, once an inventory adjustment is made.
+- :guilabel:`Scheduled Date`: the date at which a count should be made. If not otherwise specified,
+  this date will default to the 31st of December of the current year.
+- :guilabel:`User`: the person assigned to the count in the database. This can either be the person
+  physically counting the inventory, or applying the count in the database.
+- :guilabel:`Product Category`: the category assigned internally to a specific product. Unless
+  otherwise specified (i.e., as **Consumable** or **Rental**), the default *product category* is set
+  to *All*.
```

Comment:
Same with these field values, they receive :guilabel:s. Also, when referring to the field within it's definition we need to italicize and capitalize.
```suggestion
- :guilabel:`Product Category`: the category assigned internally to a specific product. Unless
  otherwise specified (i.e., as :guilabel:`Consumable` or :guilabel:`Rental`), the default *Product
  Category* is set to :guilabel:`All`.
```